product-cli/cmd/cluster/aws/create.go

```diff
@@ -0,0 +1,66 @@
+package aws
+
+import (
+	"context"
+	"github.com/spf13/cobra"
+
+	hyperv1 "github.com/openshift/hypershift/api/v1beta1"
+	hypershiftaws "github.com/openshift/hypershift/cmd/cluster/aws"
+	"github.com/openshift/hypershift/cmd/cluster/core"
+)
+
+func NewCreateCommand(opts *core.CreateOptions) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:          "aws",
+		Short:        "Creates basic functional HostedCluster resources on AWS",
+		SilenceUsage: true,
+	}
+
+	opts.AWSPlatform = core.AWSPlatformOptions{
+		AWSCredentialsFile: "",
+		Region:             "us-east-1",
```

Comment:
do want to default region? I think we shouldn't, or at least try to read it from the env variable `AWS_REGION` if it exists.

Reply:
Created [HOSTEDCP-1142](https://issues.redhat.com/browse/HOSTEDCP-1142) to capture this. 

What's the why with this though? The user can still change the region through the CLI flag.

Reply:
I think any user of our cli is most likely also using AWS cli to fulfill the perquisites and has his preferred region configured using the `AWS_REGION` variable.
so I think reading the variable instead of defaulting blindly is a more consistent UX.